requirements-dev.txt

```diff
@@ -10,6 +10,8 @@ sphinx>=3.0.0
 sphinx-rtd-theme==1.0.0
 sphinxcontrib-napoleon>=0.5.0
 sphinx-argparse
+sphinx-book-theme
+myst-parser
```

Comment:
- Suggestion: pin these requirements to an exact version (unless there's a concrete and compelling reason against that)
- Question: is `myst-parser` strictly and currently needed? (i.e. the docs build will fail if this requirement is removed)

Reply:
good idea
re:myst it will if I ever get around to using markdown, but maybe I should shelve that dream as you are suggesting